apps/typegpu-docs/src/content/examples/image-processing/ascii-filter/index.ts

```diff
@@ -0,0 +1,320 @@
+import tgpu, {
+  type Render,
+  type Sampled,
+  type TgpuBindGroup,
+  type TgpuTexture,
+} from 'typegpu';
+import * as d from 'typegpu/data';
+import * as std from 'typegpu/std';
+
+const root = await tgpu.init();
+const device = root.device;
+
+const layout = tgpu.bindGroupLayout({
+  inputTexture: { texture: 'float' },
+});
+
+const useExtendedCharacters = root['~unstable'].createUniform(d.u32);
+const displayModeBuffer = root['~unstable'].createUniform(d.u32);
+const gammaCorrectionBuffer = root['~unstable'].createUniform(d.f32);
+const glyphSizeBuffer = root['~unstable'].createUniform(d.u32, 8);
```

Comment:
Small nitpick, a more consistent naming 👀 
```suggestion
const charsetExtendedUniform = root['~unstable'].createUniform(d.u32);
const displayModeUniform = root['~unstable'].createUniform(d.u32);
const gammaCorrectionUniform = root['~unstable'].createUniform(d.f32);
const glyphSizeUniform = root['~unstable'].createUniform(d.u32, 8);
```